src/main.js

```diff
@@ -336,19 +336,35 @@ class MiniGraphCard extends LitElement {
       </div>` : '';
   }
 
+  generateLegend(entity, index) {
+    let legend = this.computeName(index);
+    const state = this.getEntityState(index);
+    const { show_legend_state } = this.config.entities[index];
```

Comment:
I am not getting this line. I believed that `show_legend_state` is an boolean option.

Reply:
It's a leftover from the previous PR
It should indeed be a bool I'll change that

Reply:
Also, could you explain a necessity of an `entity` variable here?
```
generateLegend(entity, index)
```

Reply:
It's fixed